inc/CLI/InfoCommand.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Command for printing various details about the environment.
+ *
+ * @since 3.0.0
+ *
+ * @package Required\Traduttore\CLI
+ */
+
+namespace Required\Traduttore\CLI;
+
+use Required\Traduttore\ZipProvider;
+use WP_CLI;
+use WP_CLI_Command;
+
+/**
+ * Print various details about the environment.
+ *
+ * ## OPTIONS
+ *
+ * [--format=<format>]
+ * : Render output in a particular format.
+ * ---
+ * default: list
+ * options:
+ *   - list
+ *   - json
+ * ---
+ *
+ * ## EXAMPLES
+ *
+ *     # Display various data about the Traduttore environment
+ *     $ wp traduttore info
+ *     Traduttore version:     3.0.0-alpha
+ *     WordPress version:      4.9.8
+ *     WP-CLI version:         2.0.1
+ *     WP-CLI binary path:     /usr/local/bin/wp
+ *     Git binary path:        /usr/bin/git
+ *     Mercurial binary path:  (not found)
+ *     Subversion binary path: (not found)
+ *     Cache directory:        /var/www/network.required.com/wordpress/content/traduttore
```

Comment:
```suggestion
 *     Cache directory:        /var/www/network.required.com/wp-content/traduttore
```
I would change it to a standard folder structure